src/NzbDrone.Core/Download/UsenetClientBase.cs

```diff
@@ -49,6 +49,13 @@ public override String Download(RemoteEpisode remoteEpisode)
             {
                 nzbData = _httpClient.Get(new HttpRequest(url)).ResponseData;
             }
+            catch (HttpException ex)
+            {
+                _logger.ErrorException(String.Format("Downloading nzb for episode '{0}' failed ({1})",
```

Comment:
can't we move this to download service since both `torrentClient` and `usenet` client do it?